data_model/derive/src/filter.rs

```diff
@@ -205,8 +205,6 @@ pub fn impl_filter(event: &EventEnum) -> TokenStream {
         #[derive(
             Clone,
             PartialEq,
-            PartialOrd,
```

Comment:
We don't depend on this traits anywhere in code.

Reply:
Revert. 

In library crates if you *can* implement a trait, you should.

Reply:
Reverted.